CHANGELOG.md

```diff
@@ -100,6 +102,45 @@ Branch: `nilrt/22.5/hardknott`
 - [Removed](https://github.com/ni/meta-nilrt/pull/355) boot attestation based on now dead upstream code.
 - [Removed](https://github.com/ni/meta-nilrt/pull/290) packages dropped from upstream.
 
+
+----
+
+## 8.13
+
+Branch: `nilrt/22.8/sumo`
+
```

Comment:
Would we want a summary here like the for 8.12 ("The 8.13 release is ...")?

Reply:
Will fix.